engine/apps/public_api/views/schedules.py

```diff
@@ -56,9 +56,6 @@ def get_object(self):
             raise NotFound
 
     def perform_create(self, serializer):
-        if serializer.validated_data["type"] == "web":
```

Comment:
Discussed this with @matiasb, seems like we're fine with allowing to create web shifts using public API

Reply:
If we don't want to allow to create web schedules from api, we can check User-Agent to allow to create them only from migrator. One reason to do that is because we have no any docs about creating web schedules from the public api.